src/main/java/ladder/view/InputView.java

```diff
@@ -0,0 +1,21 @@
+package ladder.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public String inputNames() {
```

Comment:
```suggestion
    public List<String> inputNames() {
```
입력 책임에 맞게 이름을 리스트로 제공해 주면 어떨까요?
그러면 비즈니스 로직에서 파싱할 필요가 없어져서 로직에만 집중할 수 있습니다.